modules/ossm-add-project-using-label-selectors-console.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-console_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the web console
+
+You can use labels selectors to add a project to the {SMProductShortName} with the {product-title} web console.
+
+.Prerequisites
```

Comment:
The list items need to use parallel grammatical contstructions -- all fragments or all complete sentences

Reply:
done